lib/src/rx_ext/serial_disposable.dart

```diff
@@ -35,8 +40,7 @@ class _SerialAlreadyDisposed extends Error {
   String toString() => "An instance of $_type has already been disposed";
 }
 
-class StreamSubscriptionSerialDisposable
-    extends SerialDisposable<StreamSubscription> {
-  StreamSubscriptionSerialDisposable()
-      : super((StreamSubscription subscription) => subscription.cancel());
+/// A serialDisable that constains an underlying stream subscription
```

Comment:
It's likely a typo: "serialDisable". Also, if it's an ID reference, use the corresponding syntax; if it's a natural language, use the corresponding formatting (separate words, capitalize as needed).

And full stop in the end of a sentence. Just imagine you're reading a magazine. Same formatting is required here.